catalog/models.py

```diff
@@ -245,9 +245,27 @@ def filter_by_products(self, products: Iterable[AbstractProduct]):
         )
 
     def get_group_tags_pairs(self) -> List[Tuple[TagGroup, List['Tag']]]:
+        """
+        Return set of group_tag pairs with specific properties.
+
+        Every pair contains tag group and sorted tag list.
+        It's sorted alphabetically or numerically.
+        Sort method depends of tag value type.
+        """
+        # @todo #STB374:120m Move tag's value to separated field.
+        #  Now we have fields like `tag.name == '10 м'`.
+        #  But should have smth like this:
+        #  `tag.value, tag.group.measure, tag.label == 10, 'м', '10 м'`.
+        #  Right now we should do dirty hacks for tags comparing mech.
```

Comment:
We have `Tag.position` field for such cases, so maybe we can use it instead. What do you think?